includes/utils.inc

```diff
@@ -54,7 +54,7 @@ function wf_crm_field_options($field, $context, $data) {
     elseif ($name == 'privacy') {
       $ret = wf_crm_get_privacy_options();
     }
-    elseif ($table === 'other') {
+    elseif ($table === 'other' || $table === 'case') {
```

Comment:
Looks to me like this will break all other pseudoconstant lookups for cases (as the `elseif ($table === 'case)` condition short-circuits all case fields.

Reading the code, I think we are actually safe to remove this outer conditional wrapping groups and tags, so we can just do `elseif ($field['table'] === 'tag') {` and `elseif ($field['table'] === 'group') {`

Reply:
Actually I have fixed this in my local but didnt update the PR. I have updated this now. Let me know if this is better or can we just remove the outer ($table === 'other') wrapper.